packages/cli/src/models/config/ConfigManager.ts

```diff
@@ -9,7 +9,6 @@ import isNil from 'lodash.isnil';
 
 const ConfigManager = {
   config: undefined,
-  cache: undefined,
```

Comment:
Where was this cache being used? I cannot find where it was read, only where it was set.

Reply:
It was used for testing telemetry only, as a way of stubbing the thing. Now that I think of it it would've been better to just stub the init function using sinon. My bad!